packages/strapi-plugin-users-permissions/controllers/Auth.js

```diff
@@ -483,6 +494,21 @@ module.exports = {
       );
     }
 
+    // Make The Username Unique
+    const username = await strapi.query('user', 'users-permissions').findOne({
```

Comment:
instead of making username as result better to make variable as boolean for example is isUsernameUnique